ansible/roles/de-TMP-RUN-ONCE-app-beta-tagger/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- debug:
+    msg: "This role should only be RUN ONCE against the metadata database. It will fail if run more than once."
+
+- name: update docker-py
+  pip: name="docker-py" state="latest"
+
+- name: pull the apps beta tagger docker image
+  docker_image: name="{{ app_beta_tagger_repo }}" tag="{{ docker_tag }}"
+
+- name: tag DE beta apps
+  shell: "docker run --rm
+    --name apps-beta-tagger
+    -v ~/.pgpass:/root/.pgpass:ro
```

Comment:
It might be better to pass the db passwords to the tool on the command line, but I had already written the tool to use `kameleon.pgpass/get-password` by the time I got to this playbook and the tool's docker container.